pyproject2setuppy/setuptools.py

```diff
@@ -0,0 +1,34 @@
+# pyproject2setup.py -- setuptools support
+# vim:se fileencoding=utf-8 :
+# (c) 2020 Eli Schwartz
+# 2-clause BSD license
+
+import os
+import tokenize
+
+def handle_setuptools(data):
+    """
+    Handle pyproject.toml unserialized into data, by ignoring it and using the
+    setuptools build system instead.
+
+    Prefer running the contents of setup.py, but fall back to running a setup()
+    function.
+    """
+    if os.path.exists('setup.py'):
+        with getattr(tokenize, 'open', open)('setup.py') as f:
+            code = f.read().replace(r'\r\n', r'\n')
+    else:
+        code = u'from setuptools import setup; setup()'
```

Comment:
I would really prefer calling `setup()` inline than this indirection.

Reply:
This was really only for consistency with the assumption of exec'ing the code in the setup.py-exists case. So if that goes in favor of os.exev, this will go too.